napari/_qt/qt_mode_buttons.py

```diff
@@ -0,0 +1,32 @@
+from qtpy.QtWidgets import QRadioButton, QPushButton
+
+
+class QtModeRadioButton(QRadioButton):
+    def __init__(self, layer, button_name, mode, tool_tip, checked=False):
```

Comment:
should some of these args be keyword based? I worry they might be hard to keep track of / remember based on order along? thoughts @jni?

Reply:
maybe made `mode=None` and `tool_tip=None` and then have them default to `button_name`, see comments below - though maybe `mode` here is more important and we want to be more primary ...

Reply:
agreed, having these as keyword-only arguments would make it much clearer as to what each parameter does in the code

Reply:
Everything that can be keyworded has been in the most recent commit, and things that are optional remain positional.  Non optional args remain positional